src/Modules/Texture/TextureLoader.cpp

```diff
@@ -47,10 +47,33 @@ std::shared_ptr<Resource> CreateSRVFromFileDDS(RenderDevice& device, RenderComma
     return res;
 }
 
+std::shared_ptr<Resource> CreateSRVFromFileKTX(RenderDevice& device, RenderCommandList& command_list, const std::string& path)
```

Comment:
What is the difference between CreateSRVFromFileDDS and CreateSRVFromFileKTX? They look exactly the same.

Reply:
Yes, they are the same. 
GLI provides classes and functions to load image files (KTX and DDS)

Reply:
So, if they are the same, what is the reason to have them? Could you please just rename CreateSRVFromFileDDS and use it for dds as well as for ktx.

Reply:
Done.